Entities/Module.php

```diff
@@ -470,6 +470,50 @@ public static function deactivateItem($slug)
         return $response;
     }
     //-------------------------------------------------
+    public static function runModuleMigrations($slug)
+    {
+
+        $module = Module::slug($slug)->first();
```

Comment:
@wri-abinash-s001 all of this code should be in `try/catch` eg:

```php
try{
    // your code
}catch (\Exception $e){
    $response = [];
    $response['success'] = false;
    if(env('APP_DEBUG')){
        $response['errors'][] = $e->getMessage();
        $response['hint'] = $e->getTrace();
    } else{
        $response['errors'][] = 'Something went wrong.';
    }
    return $response;
}
```

All your controller methods should be in `try/catch`

Reply:
ok sir I am fixing it soon